src/services/prisma-datastore/schema.prisma

```diff
@@ -0,0 +1,84 @@
+
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["extendedIndexes"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
```

Comment:
Understood. How do we inject the value of the environment variable into the schema? Will that work with `prisma generate`?  I realize it's not ideal, but it doesn't do any harm either.

Reply:
If we can make it work relatively easily then I'm all for it, but otherwise, it's not worth the effort to change, even if it is slightly architecturally unsound. Looking at `prisma generate` I don't see a way to specify the URL on the command line, which would have been better. `PrismaClient` does allow the db url to be specified.

The advantage of doing it the way it is now is that it follows the pattern laid out in all their examples, so anyone onboarding will immediately recognize what we are doing.

Reply:
https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#prismaclient.  btw, the `PrismaClient` will be injected into the `PrismaDataProvider` instance (or at the very least, the datasource URL.). 

That still leaves us with the default url in the schema which is pulled from the environment, but that's how these tools work - changing that looks to be a big lift and not worth the payback imho.

Reply:
I think it'd be better to override it in code when creating the PrismaClient 

https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#programmatically-override-a-datasource-url

Reply:
This is done. https://github.com/blues/sparrow-reference-web-app/blob/17535c0d521fc8e1f423e49bc31707312ba6dc50/src/services/ServiceLocator.ts#L67